ENIGMAsystem/SHELL/Widget_Systems/None/Makefile

```diff
@@ -1 +1,7 @@
-SOURCES += $(wildcard Widget_Systems/None/*.cpp)
+SOURCES += Widget_Systems/None/nowidget_impl.cpp
```

Comment:
Why is Android using None if it doesn't support *the only three things* that the None system **actually does?**

Reply:
@time-killer-games has offered in #1776 to work on the Android widgets.

Reply:
That wasn't my point, Robert. None used the old C console routines to offer some rudimentary functionality. Android doesn't support the console. Yet fundies spent a bunch of effort making sure that the console routines were reusable in Android. It doesn't make sense.

However... this system was already burning garbage, as it stopped being the "None" system and started being the "All things console" system. So fundies hasn't actually made it any worse... he just sort of half-assed making it work for Android. So I guess it's whatever.